boards/beagle/pocketbeagle_2/Kconfig.pocketbeagle_2

```diff
@@ -5,4 +5,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_POCKETBEAGLE_2
+	select SOC_AM6232_A53 if BOARD_POCKETBEAGLE_2_AM6232_A53
```

Comment:
We are now using AM6254. Only A0 had AM6232. A0 should be the special case, not what is currently available (A1).

Reply:
Again, only A0 is supported in this PR.